developer-docs-site/docs/reference/faq.md

```diff
@@ -0,0 +1,328 @@
+
+The Aptos devnet is a publicly deployed instance of the Aptos network, built for experimenting with new ideas. The Aptos devnet runs a set of validator test nodes. 
+
+While working with strategic partners and the developer community, we use Aptos devnet to learn where we can improve performance. We focus across all aspects of our community experience, from the Move developer and Move language to the Aptos blockchain network.
+
+### Do you have explorer for Aptos devnet?
+
+Yes. Click here: [https://explorer.devnet.aptos.dev](https://explorer.devnet.aptos.dev/)
+
+## Validators and Nodes
+
+### Can only developers run full nodes?
+
+Anyone with the proper hardware can run a node and get involved! We welcome everyone from the community to join in and participate. 
+
+See [Run a FullNode](https://aptos.dev/tutorials/full-node/run-a-fullnode) document for starting up a FullNode, or [Run a Local Testnet](https://aptos.dev/tutorials/run-a-local-testnet) document for starting a local testnet with a single validator node.
+
+### I am getting errors when I try to set up a full node. Where can I get help?
+
+See the [Troubleshooting FullNode Setup](https://aptos.dev/tutorials/full-node/troubleshooting-fullnode-setup) [p](https://aptos.dev/tutorials/full-node/troubleshooting-fullnode-setup)age in Aptos developer documentation.
+
+### What are the hardware and storage requirements to run an Aptos node?
+
+What are the minimum hardware requirements for compiling the source?
+
+We recommend 16GB of memory or more.
+
+What are the hardware requirements for a full node?
+
+See [Hardware requirements](https://aptos.dev/tutorials/full-node/run-a-fullnode#hardware-requirements) section of Aptos developer docs for running a FullNode.
+
+Can I run a node using PC with Win10 or macOS?
+
+Yes! 
+
+- For macOS or Linux: See the Aptos developer documents: [Running a local testnet](https://aptos.dev/tutorials/run-a-local-testnet) and [Run a FullNode](https://aptos.dev/tutorials/full-node/run-a-fullnode).
+- For Windows: See the presentation: [Install Aptos locally on windows](https://docs.google.com/presentation/d/12Gvf8GyRA90Gd6melO9n0zKmtCFYLry6g0ajEglXhGA/edit?usp=sharing)
+    - Here is the Chinese language version of the above document, courtesy of @aptos_cn: [https://wiki.aptos.movemove.org/jiao-cheng/yun-hang-quan-jie-dian-she-qu/windows](https://wiki.aptos.movemove.org/jiao-cheng/yun-hang-quan-jie-dian-she-qu/windows)
+
+### How do I run a full node?
+
+See a step-by-step tutorial showing how to [Run a FullNode](https://aptos.dev/tutorials/full-node/run-a-fullnode). 
+
+### How to run a full node on Google Cloud Platform?
+
+See a step-by-step tutorial showing how to [Run a FullNode on GCP](https://aptos.dev/tutorials/run-a-fullnode-on-gcp). 
+
+### Can I run a validator node?
+
+Yes. See this step-by-step tutorial describing how to [Run a Local Testnet](https://aptos.dev/tutorials/run-a-local-testnet) with a single validator node on your computer. 
```

Comment:
This seems like a non-answer, the real question is what do I need to do to run a validator, which requires having 1M of the Aptos token for mainnet, permission from AptosLabs to access for the AptosLabs testnet, no for devnet, and this link for a local network.